website/docs/working-with-containers/registries/_insecure-registry-linux.md

```diff
@@ -0,0 +1,37 @@
+#### Prerequisites
+
+- Podman.
```

Comment:
I assume it needs to be running? Before the change that was required, and there is a restart in the steps.

Reply:
It only needs to be installed. You need to restart any podman process if it exists. I will clarify.